src/libecalc/presentation/yaml/yaml_models/pyyaml_yaml_model.py

```diff
@@ -145,14 +145,27 @@ def __init__(
                 )
 
         def load(self, yaml_file: ResourceStream):
+            class UniqueKeyLoader(self.__loader):  # type: ignore[name-defined]
+                def construct_mapping(self, node, deep=False):
```

Comment:
how is deep set? Is it by us or by the SafeLoader? When we set it to false by default, does it mean that we only compare keys at a given level, or that it doesnt go further down when searching for duplicate keys?